### PR TITLE
Enable data synchronization from the UI

### PR DESCRIPTION
**Pull request by Ehevi**
As the title says +
- [x] remove the `null` row period from the list
- [x] typo fixes
- [x] some code cleanup
- [x] docker test configuration

### CODE REVIEW

File: app/public/model/PrimaryModel.js [hunk 1]
```diff
@@ -78,6 +78,14 @@ export default class PrimaryModel extends Observable {
         this.notify();
     }
 
+    async sync() {
```
Comment (xsalonx) [inline app/public/model/PrimaryModel.js:81]: users should not be able to start sync procedure, so such method in public model is unnecessary

File: docker/test.env [hunk 2]
```diff
@@ -1,26 +1,71 @@
...
+RCT_EP_ML_MC_RAW_HOST=${RCT_EP_ML_MC_RAW_HOST:-$_DEF_ML_HOST}
+RCT_EP_ML_MC_RAW_PATH=${RCT_EP_ML_MC_RAW_PATH:-/MC/?res_path=json}
+
+RCT_EP_ML_MC_DET_PROT=$_DEF_ML_PROT
+RCT_EP_ML_MC_DET_HOST=${RCT_EP_ML_MC_DET_HOST:-$_DEF_ML_HOST}
+RCT_EP_ML_MC_DET_PATH=${RCT_EP_ML_MC_DET_PATH:-/job_events.jsp?timesel=0&res_path=json}
+
+RCT_EP_ML_MC_TAG_PROT=$_DEF_ML_PROT
+RCT_EP_ML_MC_TAG_HOST=${RCT_EP_ML_MC_TAG_HOST:-$_DEF_ML_HOST}
+RCT_EP_ML_MC_TAG_PATH=${RCT_EP_ML_MC_TAG_PATH:-/MC/prodDetails.jsp?res_path=json}
```
Comment (Ehevi) [inline docker/test.env:71]: `ResProvider` fails without these

File: test/database/utilities/QueryBuilder.test.js
Comment (xsalonx) [inline test/database/utilities/QueryBuilder.test.js:1]: why in this way?

Comment (Ehevi) [reply test/database/utilities/QueryBuilder.test.js:1]: Because the structure of the files has changed. Views are stored in different files, right?

Comment (Ehevi) [reply test/database/utilities/QueryBuilder.test.js:1]: Aaah, I see what you meant. I've changed that already, thanks for noticing!

Comment (Ehevi) [reply test/database/utilities/QueryBuilder.test.js:1]: [Here](https://github.com/AliceO2Group/RunConditionTable/pull/98/files#diff-d5dfc36eca9bd37abf93a593816e6da7fd10957cab7a148f6011ea338bd18230) is the updated version

**Review by xsalonx — APPROVED** [review]

**Pull request merged**